content/ಅಂಕೆಗಳ ಅಟ್ಟಳಿಗೆ.md

```diff
@@ -0,0 +1,144 @@
+`ಪತ್+ಇಂ+ಮೂರು` = `ಪದಿಂ+ಮೂರು`(ಮೇಲೆ ನೋಡಿದ ಅಪಭ್ರಂಶ) = `ಪದಿಮ್ಮೂರು` = `ಪದಿಮೂರು` (ಲಗಂ ವರ್ಜನೆ) 
+`ಪತ್ತು+ಇಂ+ಮೂರು` = `ಪತ್ತಿಮ್+ಮೂರು` = `ಪತ್ತಿಮ್ಮೂರು` = ಪದಿಮೂರು (ಅಪಭ್ರಂಶ)
+ಈ ರೂಪಗಳೇ ಮೂಲರೂಪವಿರಬಹುದು.
+**ಪಂಚಮೀ ವಿಭಕ್ತಿಯಿಂದ ಹುಟ್ಟಿದುದೇ ಕನ್ನಡ ಸಂಖ್ಯೆಗಳಲ್ಲಿರುವ ಸ್ವಾರಸ್ಯ**
+
+### ಪದಿಂ, ಪಯಿಂ, ಪನ್
+ಈ ಮೂರು ಪದಗಳೂ **ಪತ್ತಿಂ** ನ ವಿವಿಧ ಅಪಭ್ರಂಶ ರೂಪಗಳು ಎಂಬುದು ಈ ಲೇಖನದ ಮತ.
+ಆ ಕಾಲದ ಪ್ರಯೋಗಗಳನ್ನು ಗಮನಿಸಿ ಶಬ್ದಮಣಿದರ್ಪಣದಲ್ಲಿ ಕೇಶಿರಾಜನು ಇವುಗಳನ್ನು **ಪತ್ತು**ವಿನ ಆದೇಶರೂಪಗಳು ಎಂದಷ್ಟೇ ಹೇಳಿದ್ದಾನೆ. ಯಾಕೆ ಎಂಬುದನ್ನು ವಿಶ್ಲೇಷಿಸಿಲ್ಲ.
+ಆದರೆ ಮೇಲೆ ನೋಡಿದಂತೆ `ಪದಿಂ` `ಪತಿಂ` ನ ಅಪಭ್ರಂಶ. `ಪದಿಂ` ನ ದಕಾರ ಶಿಥಿಲವಾಗಿ `ಪಯಿಂ`. ಅನುಸ್ವಾರದ ನಕಾರದಿಂದ `ಪಯಿನ್` ಬಂದು ಅದರ ಯಕಾರ ಲೋಪವಾಗಿ `ಪನ್` ಬಂದಿರಬೇಕು. 
+
+
+>ಪತ್ತರ್ಕೆ ಪನ್ನೆನಿಕ್ಕುಮ
+ದುತ್ತರಕೊಂದೆರಡು ನೆಲಸೆ ಪದಿಯೆಂದಕ್ಕುಂ
+ಮತ್ತೊದವಿ ಮೂಱು ನಾಲ್ಕಿರೆ
+ಸುತ್ತೇಂ ಸ್ವರಮಿರೆ ನಕಾರವಿಧಿ ಮುಂದೆಲ್ಲಂ ||[ಸೂತ್ರ 203](https://youtu.be/ZIjoq7hUcVc?list=PLYc_H2yt-vAJhMjmz3cztfjiNO7XhgRQG&t=3365)||
+
+`ಪತ್`-ಇಗೆ ವಿಭಕ್ತಿಸೂಚಕವಾದ ಅನುಸ್ವಾರ ಸಮೇತವಾದ ಪದಿಂ ಇತ್ಯಾದಿಗಳು ಆದೇಶವಾಗಲಾರದು [ಅನುಸ್ವಾರದ ಅನುಸಾರ ನೋಡಿ](ವಕಾರದ%20ವಿಭಕ್ತಿ.md#ದ್ವಿತ್ವಸಂಧಿ). `ಪತಿಂ` ಅಥವಾ `ಪತ್ತಿಂ` ಗೆ ವಿಭಕ್ತರೂಪದ ಅಂದರೆ `ಪದಿಂ` -ನ ಆದೇಶವಾಗಿರಬಹುದು ಅಷ್ಟೇ.
+ಅದಲ್ಲದೇ, ಅನುಸ್ವಾರವು ಮಕಾರ ನಕಾರ ಗಳ ಉಚ್ಚಾರಣೆಗಳನ್ನು ಧರಿಸುವುದನ್ನು ಬೇರೆಡೆಯೂ [ನೋಡಿದ್ದೇವೆ](ಅನುಸ್ವಾರದ%20ಅನುಸಾರ.md#ಅರ್ಧಾನುಸ್ವಾರದ-ಜಾಡಿನಲ್ಲಿ). 
+
+## ಲಗಂ ವರ್ಜನೆ
+ಕನ್ನಡ ಭಾಷೆಯು ಶಬ್ದದಾರಂಭದಲ್ಲಿ ಲಗಂ ಜತಿಯನ್ನು ವರ್ಜಿಸುತ್ತದೆ. ಇದರ ಬಗ್ಗೆ [ಇಲ್ಲಿ](ಅನುಸ್ವಾರದ%20ಅನುಸಾರ.md#ಅರ್ಧಾನುಸ್ವಾರದ-ಜಾಡಿನಲ್ಲಿ) ಓದಬಹುದು. ಜಗಣ ಅಥವಾ `ಲಗಂ` ವರ್ಜಿಸಲು ಕೆಲವು ಆಗಮ ಆದೇಶ ಲೋಪಗಳು ದ್ವಿತ್ವಗಳು ಉಂಟಾಗುತ್ತವೆ.
+
+# ಅಂಕಿಗಳ ಅಟ್ಟಣೆ
+1. ಕನ್ನಡದಲ್ಲಿ ಜಗಣ/ಲಗಂ ವರ್ಜ್ಯ
+2. ಹಳಗನ್ನಡದಲ್ಲಿ 10 = ಪತ್
+3. ಇಂ ಪ್ರತ್ಯಯ ಪಂಚಮೀ ವಿಭಕ್ತಿಯ ಸಾಪೇಕ್ಷ ದೂರವನ್ನು ಕೊಡುತ್ತದೆ
+4. ಅನುಸ್ವಾರವು ಮಕಾರ ನಕಾರ ಗಳೆರಡನ್ನೂ ಹೊಂದಬಲ್ಲುದು.
+ಈ ಮೇಲಿನ ಅಂಶಗಳನ್ನು ಮನದಲ್ಲಿಟ್ಟುಕೊಂಡು 10 - 18 ಅಂಕಿಗಳ ಅಟ್ಟಿಯನ್ನು ಮಾಡಬಹುದು.
+
+## ಹನ್ನೊಂದು
+ಪತ್ತಿಂ+ಒಂದು ->ಪದಿಂ+ಒಂದು -> ಪಯಿನ್+ಒಂದು -> ಪನ್+ಒಂದು -> ಪನೊಂದು -> ಪನ್ನೊಂದು (ಲಗಂ ವರ್ಜನೆ, ನಾನ ನಾನಾ ಜತಿಗಾಗಿ)
+
+## ಹನ್ನೆರಡು
+ಪತ್ತಿಂ+ಎರಡು ->ಪದಿಂ+ಎರಡು -> ಪಯಿನ್+ಎರಡು -> ಪನ್+ಎರಡು -> ಪನೆರಡು -> ಪನ್ನೆರಡು ( ನಾನ ನಾನಾ ಜತಿಗಾಗಿ)
+
+## ಹದಿಮೂರು
+ಪತ್ತಿಂ+ಮೂರು ->ಪದಿಂ+ಮೂರು -> ಪದಿಮ್+ಮೂರು -> ಪದಿಮ್ಮೂರು ->ಪದಿಮೂರು (ಲಗಂ ವರ್ಜನೆ)
+
+## ಹದಿನಾಲ್ಕು
+ಪತ್ತಿಂ+ನಾಲ್ಕು ->ಪದಿಂ+ನಾಲ್ಕು -> ಪದಿನ್+ನಾಲ್ಕು (ಅನುಸ್ವಾರದ ನಕಾರ ಉಚ್ಚಾರಣೆ) -> ಪದಿನ್ನಾಲ್ಕು ->ಪದಿನಾಲ್ಕು (ಲಗಂ ವರ್ಜನೆ)
+
+## ಹದಿನೈದು
+ಪತ್ತಿಂ+ಐದು ->ಪದಿಂ+ಐದು -> ಪದಿನ್+ಐದು (ಅನುಸ್ವಾರದ ನಕಾರ ಉಚ್ಚಾರಣೆ) -> ಪದಿನೈದು
+
+## ಹದಿನಾರು
+ಪತ್ತಿಂ+ಆರು ->ಪದಿಂ+ಆರು -> ಪದಿನ್+ಆರು (ಅನುಸ್ವಾರದ ನಕಾರ ಉಚ್ಚಾರಣೆ) -> ಪದಿನಾರು
+
+## ಹದಿನೇಳು
+ಪತ್ತಿಂ+ಏಳು ->ಪದಿಂ+ಏಳು -> ಪದಿನ್+ಏಳು (ಅನುಸ್ವಾರದ ನಕಾರ ಉಚ್ಚಾರಣೆ) -> ಪದಿನೇಳು
+
+## ಹದಿನೆಂಟು
+ಪತ್ತಿಂ+ಎಂಟು ->ಪದಿಂ+ಎಂಟು -> ಪದಿನ್+ಎಂಟು (ಅನುಸ್ವಾರದ ನಕಾರ ಉಚ್ಚಾರಣೆ) -> ಪದಿನೆಂಟು
+
+## ಹತ್ತೊಂಬತ್ತು +
+ಇಲ್ಲಿಂದ ಮುಂದೆ ಷಷ್ಠೀ ತತ್ಪುರುಷ ಸಮಾಸಗಳೇ ಕಾಣುತ್ತವೆ, ಪಂಚಮೀ ಇಲ್ಲ
+ಪತ್ತ+ಒಂಬತ್ತು -> ಪತ್ತೊಂಬತ್ತು
+ಪತ್ತರ+ಒಂಬತ್ತು -> ಪತ್ತ್ರೊಂಬತ್ತು ಉತ್ತರ ಕರ್ನಾಟಕದಲ್ಲಿ ಪ್ರಯೋಗದಲ್ಲಿದೆ
+ಇರ್+ಪತ್+ಅ +ಒಂದು -> ಇರ್ಪತ್ತೊಂದು -> ಇಪ್ಪತ್ತೊಂದು ರಲ್ಲಿ ಕೂಡಾ ಷಷ್ಠೀ ವಿಭಕ್ತಿಯ `ಅ` ಪ್ರತ್ಯಯ ಕಾಣುತ್ತದೆ. ಇಪತ್ತೊಂದನ್ನು ಇಪ್ಪತ್ತು+ಒಂದು ಎಂದು ಬಿಡಿಸುವುದು ಸರಿಯಾಗದು, ಇಪ್ಪತ್ತ ಒಂದು ಎಂಬ ಪ್ರಯೋಗ ಆಡುಮಾತಿನಲ್ಲಿ ಇದ್ದೇ ಇದೆ. ನೂರ ಇಪ್ಪತ್ತೊಂದು ರಲ್ಲಿ ನೂರೂ ಇಪ್ಪತ್ತೂ ಷಷ್ಠೀ ವಿಭಕ್ತಿಯಲ್ಲಿರುವುದು ಇದಕ್ಕೆ ಪುಷ್ಠಿ ಕೊಡುತ್ತದೆ. ಪಂಚಮಿಯ ಕೌತುಕ ಹನ್ನೊಂದರಿಂದ ಹದಿನೆಂಟರವರೆಗೆ ಮಾತ್ರ. ಬಹುಶಃ ಆಕಾಲದಲ್ಲಿ ಇಪ್ಪತ್ತರೊಳಗಿನ ಅಂಕಿಗಳು ಸಾಕಾಗಿದ್ದವೇನೋ.
```

Comment:
> ಬಹುಶಃ ಆಕಾಲದಲ್ಲಿ ಇಪ್ಪತ್ತರೊಳಗಿನ ಅಂಕಿಗಳು ಸಾಕಾಗಿದ್ದವೇನೋ.

Don't be so quick to assume your ancestor's incompetence😅 There are other possible reasons. Ex. is ಇರ್ಪದಿಮೂರು 20 + 3 or 2*13? This ambiguity exists in till 99. Hence, 6 instead of 5 from 20 onwards. I can't think of anything for 6 in ಹತ್ತೊಂಬತ್ತು. Once 6 is set from 19-99. Inertia is a good enough explanation for 100 onwards. Does is make sense to give this explanation instead?

Reply:
once could argue that ಇರ್ಪತ್ +ಇಂ+ಒಂದು after 6 samaasa and pratyaya drop ಇರ್ಪತ್+ಒಂದು = ಇರ್ಪತೊಂದು

Reply:
added that inertia line.